include/boost/json/impl/monotonic_resource.ipp

```diff
@@ -10,61 +11,110 @@
 #ifndef BOOST_JSON_IMPL_MONOTONIC_RESOURCE_IPP
 #define BOOST_JSON_IMPL_MONOTONIC_RESOURCE_IPP
 
+#include <algorithm>
+
 #include <boost/json/monotonic_resource.hpp>
 
 namespace boost {
 namespace json {
 
-struct monotonic_resource::block
+// ensures that the alignment of top is
+// the strictest fundamental alignment requirement
+struct alignas(alignof(std::max_align_t)) 
```

Comment:
Is this C++11?

Reply:
https://en.cppreference.com/w/cpp/language/alignas